service/org.nemomobile.BtrfsBalancer.service

```diff
@@ -0,0 +1,5 @@
+[D-BUS Service]
+Interface=org.nemomobile.BtrfsBalancer
+Name=org.nemomobile.BtrfsBalancer
+Exec=/usr/sbin/btrfs-balancer server
+User=root
```

Comment:
I'm not so sure about the "After" entry (and ssu doesn't have it either), because we don't want the service to start automatically at booting.